ocabundles.json

```diff
@@ -6,9 +6,9 @@
 "M6dhuFj5UwbhWkSLmvYSPc:2:member_card:1.53": { "path": "OCABundles/schema/bcgov-digital-trust/member-card-showcase/OCABundle.json" },
 "QEquAHkM35w4XVT3Ku5yat:2:member_card:1.53": { "path": "OCABundles/schema/bcgov-digital-trust/member-card-showcase/OCABundle.json" },
 "AuJrigKQGRLJajKAebTgWu:3:CL:209526:default": { "path": "OCABundles/schema/bcgov-digital-trust/member-card-showcase/OCABundle.json" },
-"S7S2wzcF2giKuwxdeLBk69:2:mines-act-permit:0.3": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit-0_3/OCABundle.json" },
-"S7S2wzcF2giKuwxdeLBk69:2:bc-mines-act-permit:1.1.1": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit-1_1_1/OCABundle.json" },
-"A2UZSmrL9N5FDZGPu68wy:2:bc-mines-act-permit:1.1.1": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit-1_1_1/OCABundle.json" },
+"S7S2wzcF2giKuwxdeLBk69:2:mines-act-permit:0.3": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit/0.3/OCABundle.json" },
+"S7S2wzcF2giKuwxdeLBk69:2:bc-mines-act-permit:1.1.1": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit/1.1.1/OCABundle.json" },
+"A2UZSmrL9N5FDZGPu68wy:2:bc-mines-act-permit:1.1.1": { "path": "OCABundles/schema/bcgov-digital-trust/mines-act-permit/1.1.1/OCABundle.json" },
```

Comment:
Is this supposed to be duplicated?

Reply:
What are you saying is duplicated — not sure.  Is it that there are two 1.1.1 entries?  That is because the README.md in that folder has the two entries.  The script takes whatever identifiers you define in the README.md table and adds them to this file — pointing at the corresponding bundle.

Reply:
Ok, it's deliberately doing that for each identifiers. Excellent!